packages/clay-panel/src/index.tsx

```diff
@@ -135,6 +134,7 @@ function ClayPanel({
 			{collapsable && (
 				<>
 					<ClayButton
+						aria-controls="panel-body"
```

Comment:
The value of `aria-controls` needs to be the id of the element it controls.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

Reply:
fixed!